logging/google/cloud/logging/handlers/transports/sync.py

```diff
@@ -29,7 +33,7 @@ class SyncTransport(Transport):
     def __init__(self, client, name):
         self.logger = client.logger(name)
 
-    def send(self, record, message):
+    def send(self, record, message, resource=_GLOBAL_RESOURCE):
```

Comment:
if you make `resource` a optional/kwarg here, you should also make it an optional/kwarg in `base.py` so that you preserve the signature.